cli/CHANGELOG.md

```diff
@@ -17,7 +21,7 @@ _Released 09/12/2023_
 
 **Bugfixes:**
 
-- Edge cases where `cy.intercept()` would not properly intercept and asset response bodies would not properly be captured for test replay have been addressed. Addressed in [#27771](https://github.com/cypress-io/cypress/pull/27771).
+- Edge cases where `cy.intercept()` would not properly intercept and asset response bodies would not properly be captured for Test Replay have been addressed. Addressed in [#27771](https://github.com/cypress-io/cypress/pull/27771).
```

Comment:
Just pointing out that these changes need to be made [here](https://github.com/cypress-io/cypress-documentation/blob/main/docs/guides/references/changelog.mdx#L15) to actually be applied.

Reply:
https://github.com/cypress-io/cypress-documentation/pull/5487